templates/webapi/test/infopanel.html.ep

```diff
@@ -4,6 +4,7 @@
                 % if (current_route 'latest') {
                     <%= link_to $job->id => url_for ('test', testid => $job->id) %>:
                 % }
+                <i>#<a class="copy-jobid" href data-jobid="<%= $job->id %>"><%= $job->id %></a></i>
```

Comment:
```suggestion
                <i><a class="copy-jobid" href data-jobid="<%= $job->id %>">#<%= $job->id %></a></i>
```

Looks better and makes more sense if hash is part of the id.
Also you could set the href to link to the current job.